src/utils/fetchQuickstarts.ts

```diff
@@ -18,22 +18,30 @@ export interface ExtendedQuickstart extends QuickStart {
   metadata: TaggedMetadata;
 }
 
+export type FetchQuickstartsOptions = {
+  'product-families'?: string[];
+  content?: string[];
+  'use-case'?: string[];
+};
+
 async function fetchQuickstarts(
   getUser: ChromeAPI['auth']['getUser'],
-  bundle?: string
+  options?: FetchQuickstartsOptions
 ) {
   const user = await getUser();
   if (!user) {
     throw new Error('User not logged in');
   }
-
   const account = user.identity.internal?.account_id;
 
   const quickstartsPath = `${API_BASE}/${QUICKSTARTS}`;
 
   const contentPromise = axios
     .get<{ data: { content: ExtendedQuickstart }[] }>(quickstartsPath, {
-      params: { account, bundle },
+      params: {
+        account,
+        ...options,
```

Comment:
@JetyAdam just FYI I see this locally when trying out your changes
![image](https://github.com/user-attachments/assets/9df83f3d-2844-4fdd-accf-f43fe5a26c8e)


Note the `//` after `v1` before `quickstarts`. I don't think that is related to your changes - but we should take a todo to investigate.

Reply:
That's weird. I'll create a Jira ticket for that

Reply:
@JetyAdam thanks. If you could link it here I'd appreciate it!

Reply:
Sure thing. Here is the link: https://issues.redhat.com/browse/RHCLOUD-35438